code/style.css

```diff
@@ -1,7 +1,318 @@
+  right: 10%;
+  color: white;
+}
+.tagline h3 {
+  font-family: "Merriweather", serif;
+  font-style: italic;
+  font-size: 43px;
+  margin-bottom: 10px;
+}
+.tagline p {
+  margin-bottom: 10px;
+  font-size: 18px;
+}
+.tagline button {
+  padding: 0.5em 3em;
+  border: 0.16em solid #ffffff;
+  margin: 0 0.3em 0.3em 0;
+  text-decoration: none;
+  text-transform: uppercase;
+  color: #ffffff;
+  text-align: center;
+  transition: all 0.15s;
+  background-color: rgba(96, 87, 155, 0.205);
+  border-radius: 15px;
+}
+.tagline button:hover {
+  color: #c1b3cc;
+  border-color: #9f94a8;
+}
+/* Here I wanted to do a JS button transition to the Newsletter section but I didn't manage to solve it this week */
+.tagline button:active {
+  color: #c1b3cc;
+  border-color: #9f94a8;
+}
+.main {
+  width: 100%;
+  display: grid;
+  grid-template-columns: repeat(2, minmax(0, 1fr));
+}
+.faq,
+.newsletter {
+  background-color: white;
+  width: 80%;
+  max-height: 80%;
+  margin: auto;
+  padding: 20px;
+  overflow: scroll;
+  box-shadow: 2px 2px 4px #525f4d;
+}
+.faq h2 {
+  font-family: "Merriweather", serif;
+  font-style: italic;
+  font-size: 30px;
+  color: #3e3f4b;
+  padding-bottom: 10px;
+}
+/* Accordion styling */
+button.accordion {
+  width: 100%;
+  border: none;
+  outline: none;
+  text-align: left;
+  padding: 15px 20px;
+  font-size: 16px;
+  color: #333;
+  transition: background-color 0.2s linear;
+  margin-bottom: 3px;
+}
+button.accordion:after {
+  content: "+";
+  font-size: 18px;
+  float: right;
+}
+button.accordion.is-open:after {
+  content: "-";
+}
+button.accordion:hover,
+button.accordion.is-open {
+  background-color: rgb(217, 211, 224);
+}
+.accordion-content {
+  background-color: rgb(233, 229, 240);
+  font-size: 14px;
+  color: #3e3f4b;
+  width: 100%;
+  padding: 0 20px;
+  margin-bottom: 10px;
+  max-height: 0;
+  overflow: hidden;
+  transition: max-height 0.2s ease-in-out;
+}
+/* Accompanying image */
+.faq-image,
+.newsletter-image {
+  background-color: #f1ecec;
+}
+.faq-image img {
+  width: 80%;
+  height: 100%;
+  object-fit: cover;
+  margin: auto;
+  display: block;
+  padding: 50px;
+  opacity: 0.7;
+}
+/* Big quote styling */
+.quote {
+  grid-column-start: 1;
+  grid-column-end: 3;
+  background-color: #889189ad;
+  display: block;
+  align-content: center;
+  padding: 10vh;
+}
+.quote h3 {
+  font-size: 40px;
+  color: #f1ecec;
+  font-style: italic;
+  text-align: center;
+  font-family: "Abril Fatface";
+  font-weight: 100;
+}
+.quote h2 {
+  text-align: center;
+  color: rgb(216, 206, 233);
+  text-transform: uppercase;
+  font-size: 16px;
+}
+/* Newsletter signup section styling */
+.newsletter {
+  margin: auto;
+  padding: 20px;
+  background-color: white;
+  width: 60%;
+  max-height: 80%;
+  overflow: scroll;
+  box-shadow: 2px 2px 4px #525f4d;
+}
+.newsletter-image img {
+  /* Here I flipped the image horizontally to fit the design better */
+  -webkit-transform: scaleX(-1);
+  transform: scaleX(-1);
```

Comment:
Ahaa, at first I did not understand this but when inspecting it in dev tool it made sense. :) As so many other things on your page, this was a really nice touch and I totally agree how it better fits the overall design. I like how much thought you have given the design aspect, it's absolutely beautiful and everything is really nicely put together. :)